src/pages/MetricDetail.svelte

```diff
@@ -2,10 +2,116 @@
   import { getMetricData } from "../state/api";
 
   export let params;
+
   const metricDataPromise = getMetricData(params.app, params.metric);
 </script>
 
 {#await metricDataPromise then metric}
   <h1>{metric.name}</h1>
-  <p>{metric.description}</p>
+  <h2>{metric.description}</h2>
+  <div />
+  <section id="metric-detail-view">
+    <div>
+      <footer>
+        <div>
+          <div>
+            <dl>
```

Comment:
If we're going to use a description list here, we would want something like:

```svelte
<dl>
<dt>Relevant Bugs</dt>
{#each metrics.bugs as bug, i}
 <dd><a href={bug} ...>{i+1}</a></dd>
```

For more information see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

Reply:
Yes ! I was trying to do the same :) Description List was trying to keep inline with older netlify app. But given that all the formatting had not been added, I have minimalized it to a simple Table. 
Thanks !